prep-css-display/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Display Properties</title>
+  <style>
+
+  </style>
+</head>
+
+<body>
+  <div style = "border: 4px solid blue;">I am a div and by default I am block, which means I take up all the space I can horizontally.</div>
+
+  <div
+    span {
+      height: 100px;, width: 100px;
+    }></div>
```

Comment:
Hi Derek, for this exercise we want to put all of our CSS code between the `<style></style>` tags in the `<head></head>` of the document rather than doing inline styling.